src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
+
+    client.evaluate(`${inputString}.toString()`, { frameId }).then(res => {
```

Comment:
I think we should get rid of toString

Reply:
Not quite sure how to catch execution output (we evaluate function return with `()`), function body text (it is a function without the `()`) or variable content at the same time a different way.